cost/turbonomics/credential_refresh/turbonomic_cred_refresh.pt

```diff
@@ -0,0 +1,166 @@
+name "Turbonomic Credential Refresh"
+rs_pt_ver 20180301
+type "policy"
+short_description "A policy that refreshes the cookie used to authenticate with Turbonomic APIs"
+severity "low"
+category "Operational"
+default_frequency "hourly"
+info(
```

Comment:
Looks like the fields for this info block are missing, in this PR are various examples of the values these parameters can take: https://github.com/flexera-public/policy_templates/pull/1309/files

![image](https://github.com/flexera-public/policy_templates/assets/54189123/6d53b1bd-6a3b-4193-b991-b799849bbf53)